module2/projects/relational_rails.md

```diff
@@ -314,69 +309,94 @@ I should be taken to that parents edit page where I can update its information j
 ```
 [ ] done
 
-User Story 19, Parent Delete From Parent Index Page (x1)
+User Story 18, Child Update From Childs Index Page (x1)
 
 As a visitor
-When I visit the parent index page
-Next to every parent, I see a link to delete that parent
+When I visit the `child_table_name` index page or a parent `child_table_name` index page
+Next to every child, I see a link to edit that child's info
 When I click the link
-I am returned to the Parent Index Page where I no longer see that parent
+I should be taken to that `child_table_name` edit page where I can update its information just like in User Story 11
 ```
 
+**Iterations 1 and 2 will be reviewed at your second check-in**
```

Comment:
Can we change this to "Iteration 1 will be reviewed at your second checkin"? I don't think we would have time to check both iterations and I think just looking at iteration 1 will give us good insight into how the students are doing, as well as get them feedback on what they need to work on.